recipes-connectivity/prplmesh/prplmesh.bb

```diff
@@ -0,0 +1,53 @@
+inherit cmake pythonnative
+
+S = "${WORKDIR}/git"
+OECMAKE_SOURCEPATH = "${S}/"
+
+SECTION = "net"
+LICENSE = "BSD-2-Clause-Patent"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=7b45146d47e73bcbac1068e5cfc2a9fb"
+DEPENDS = "python-native python-pyyaml-native json-c readline openssl"
+RDEPENDS_${PN} = "iproute2 busybox"
+SUMMARY = "prplMesh"
+
+SRCREV = "911392026280d9961823a9359e6cc4f08c9ee0a2"
+SRC_URI = "git://github.com/prplfoundation/prplMesh.git;protocol=http;branch=master"
+
+INSANE_SKIP_${PN} = "useless-rpaths"
```

Comment:
I would love our rpaths to be sane so we don't need this :-)

Reply:
I remove this `INSANE_SKIP`
On one of intermediate states of recipe Yocto prints QA check error without this statement.
No recipe works without it.